subt_ros/launch/vehicle_topics.launch

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0"?>
 <launch>
+  <env name="IGN_TRANSPORT_TOPIC_STATISTICS" value="1"/>
```

Comment:
Is this needed here if it's set in the `.ign` launch files? If so, we probably need to set it in each submitted model's `vehicle_topics.launch` file.

Reply:
Yup, not needed and removed in 96e10d03b5a6a22cee47adcb847ba4faf3067749